account_invoice_pricelist/model/account_invoice_line.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# © 2015-2016 GRAP <http://www.grap.coop>.
+# © 2017 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later <http://www.gnu.org/licenses/agpl.html>.
+from openerp import models, api
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    @api.multi
+    def product_id_change(
+            self, product_id, uom_id, qty=0, name='', type='out_invoice',
+            partner_id=False, fposition_id=False, price_unit=False,
+            currency_id=False, company_id=None):
+        """Make sure pricelist is used on product.
+
+        NB: On original method argument product_id is confusingly called
```

Comment:
don't do this. This way, you actually change the function's signature, because it's perfectly valid to pass keyword arguments by name